packages/page-accounts/src/modals/ProxyOverview.tsx

```diff
@@ -147,22 +147,22 @@ function NewProxy ({ index, onChangeAccount, onChangeType, onRemove, proxiedAcco
   );
 }
 
-function getProxyTypeInstance (api: ApiPromise, index = 0): NodeRuntimeProxyType {
+function getProxyTypeInstance(api: ApiPromise, index = 0): NodeRuntimeProxyType {
```

Comment:
```suggestion
function getProxyTypeInstance (api: ApiPromise, index = 0): NodeRuntimeProxyType {
```

Fix, CI fails linting due to this.